projects/angular/src/forms/styles/_toggles.clarity.scss

```
@@ -46,6 +49,9 @@
       width: toggle-variables.$clr-toggle-switch-base-width;
       flex: 0 0 auto;
       border-radius: tokens.$cds-alias-object-border-radius-200; // Style outline radius properly
+      background-color: toggle-variables.$clr-toggle-bg-color-off;
+      @include clr-toggle-transition-styles;
+      transition-property: border-color, background-color;
```

Same here -- we don't have e border on the input so this transition is not needed and also duplicated between here and the ::checked state. Do we need it on both places if it's the same?
```suggestion
      transition-property: background-color;
```

---

Corrected in commit dc61037.